projects/addon-doc/components/demo/index.ts

```diff
@@ -42,11 +44,21 @@ import {skip} from 'rxjs';
 
 const MIN_WIDTH = 160;
 
+@Pipe({name: 'json'})
```

Comment:
Should we file a bug into Angular to support it in default pipe?

Reply:
Replacer having a perf impact, I wouldn't add it to the default implementation of the JsonPipe.